### ISSUE TITLE
Can't use 0.7.0 with rspec 3.5.0.beta1

### ISSUE THREAD

**Issue by justin808** [kucaahbe/rspec-html-matchers#42]
After running bundle update, bundler insists on using an older version of this gem.

```
Installing rspec-support 3.5.0.beta1 (was 3.4.1)
Installing rspec-mocks 3.5.0.beta1 (was 3.4.1)
Using rspec-activemodel-mocks 1.0.3
Installing rspec-expectations 3.5.0.beta1 (was 3.4.0)
Using rspec-collection_matchers 1.1.2
Installing rspec-core 3.5.0.beta1 (was 3.4.2)
Installing rspec 3.5.0.beta1 (was 3.4.0)
Installing rspec-rails 3.5.0.beta1 (was 3.4.0)
Installing rspec-html-matchers 0.6.0 (was 0.7.0)
Bundle updated!
```

**Comment by justin808** [comment]
```
bundle update rspec-html-matchers                                                                       1 ↵ ✹ ✭ [14:33:48]
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies........
Bundler could not find compatible versions for gem "rspec-core":
  In Gemfile:
    rspec-html-matchers (= 0.7.0) was resolved to 0.7.0, which depends on
      rspec (~> 3) was resolved to 3.0.0, which depends on
        rspec-core (~> 3.0.0)

    rspec-rails (= 3.5.0.beta1) was resolved to 3.5.0.beta1, which depends on
      rspec-core (= 3.5.0.beta1)
```

**Comment by kucaahbe** [comment]
hello @justin808 thanks for adding issue, you could try 0.7.1 version from master branch. Release to rubygems going to happen a bit later, after [tests pass](https://travis-ci.org/kucaahbe/rspec-html-matchers/builds/109951024).
